### DISCUSSION TITLE
Change Apollo Sandbox's server endpoint ?

### DISCUSSION THREAD

**Discussion by alexandre-ducarne** [apollographql/apollo-server#7764] [Q&A]
Hi, I've just migrated from Apollo 2 to Apollo 4 and I have a question linked to Apollo Sandbox.
Before we had an API hosted on api.mywebsite.com
It had multiple paths like
- `/docs` for a generated doc about the endpoint
- `/docs/playground` for the Grahql Playground that our clients can freely use

Now that we have migrated to Apollo 4 I don't see any way to set the default server endpoint the Apollo Sandbox should connect to (it always defaults to the same URI as the one it is hosted on) : 
We want to keep hosting the playground at `/docs/playground` but the graphql endpoint it targets by default is `/docs/playground` whereas our graphql API is at `/graphql`.
I tried to check the different inputs the ApolloServerPluginLandingPageLocalDefault can take but I don't see anything related to server endpoint URIs

Thanks in advance !

EDIT: For now our workaround is to redirect immediately when landing on the previous url like so : 
```typescript
app.use("/docs/playground", (_, res)=> {res.redirect("/graphql")})
```

EDIT2: Found the solution [here](https://www.apollographql.com/docs/graphos/explorer/sandbox/#url-parameters). While it's good we can provide a query param, it'd be great to be able to set it from the plugin's args

**Comment by trevor-scheer** [comment]
Thanks for the feedback and follow-up with your solution @alexandre-ducarne. I've passed this on the the team responsible for Sandbox/Explorer.
